public.js

```diff
@@ -66,6 +65,7 @@ var init = function(app, schema, sequelize, config) {
                   if (/testshare/.test(pathname)) {
                     res.render('shareheaders', {
                       'extraProperties': domainInfo.extraProperties || [],
+                      'extraMeta': domainInfo. extraMeta || [],
```

Comment:
Again, twitterMeta is more specific (leaving room to implement more social media in the future)